resources/zib/zib-CareTeam.xml

```diff
@@ -0,0 +1,173 @@
+      <path value="CareTeam.participant" />
+      <slicing>
+        <discriminator>
+          <type value="profile" />
+          <path value="member.resolve()" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CareTeam.participant:contactperson">
+      <path value="CareTeam.participant" />
+      <sliceName value="contactperson" />
+    </element>
+    <element id="CareTeam.participant:contactperson.member">
+      <path value="CareTeam.participant.member" />
+      <short value="CareTeamMember" />
+      <definition value="A contact person of the patient as member of the care team." />
+      <alias value="Contactpersoon" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+      </type>
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.8" />
+        <comment value="CareTeamMember" />
+      </mapping>
+    </element>
+    <element id="CareTeam.participant:patient">
+      <path value="CareTeam.participant" />
+      <sliceName value="patient" />
+    </element>
+    <element id="CareTeam.participant:patient.member">
+      <path value="CareTeam.participant.member" />
+      <short value="CareTeamMember" />
+      <definition value="The patient as member of the care_team." />
+      <alias value="Patient" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Patient" />
+      </type>
+      <mapping>
+        <identity value="zib-careteam-v1.0-2020EN" />
+        <map value="NL-CM:17.3.7" />
+        <comment value="CareTeamMember" />
+      </mapping>
+    </element>
+    <element id="CareTeam.participant:healthcareProfessional">
+      <path value="CareTeam.participant" />
+      <sliceName value="healthcareProfessional" />
+      <max value="1" />
+    </element>
+    <element id="CareTeam.participant:healthcareProfessional.role">
+      <path value="CareTeam.participant.role" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CareTeam.participant:healthcareProfessional.role:healthProfessionalRole">
+      <path value="CareTeam.participant.role" />
+      <sliceName value="healthProfessionalRole" />
+      <short value="HealthProfessionalRole" />
+      <definition value="The role the health professional fulfils in the healthcare process. For health professionals, this could be for example attender, referrer or performer." />
+      <alias value="ZorgverlenerRol" />
+      <max value="1" />
+      <binding>
+        <strength value="required" />
+        <description value="The role the health professional fulfils in the healthcare process." />

```

Comment:
I thought we didn't need to fill in the description of a value list?

Reply:
We don't need to, but we can populate it when appropriate. I think this is a useful addition.